server/container_create_linux.go

```diff
@@ -714,7 +714,7 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		mountLabel,
 		containerInfo.RunDir,
 		s.config.DefaultMountsFile,
-		containerInfo.RunDir,
+		runtimeCwd,
```

Comment:
looking at the code, containerCwd seems a better fit (as it falls back to the image's cwd if it's not specified by the kubelet)... thoughts?